starfyre/router.py

```diff
@@ -0,0 +1,58 @@
+import os
+from pathlib import Path
+
+class Router:
```

Comment:
Let's call it file router as we will have an app router in the future

Reply:
Yes, thanks!